erpnext/accounts/doctype/journal_entry/journal_entry.js

```diff
@@ -347,8 +347,8 @@ cur_frm.cscript.update_totals = function(doc) {
 	var td=0.0; var tc =0.0;
 	var accounts = doc.accounts || [];
 	for(var i in accounts) {
-		td += flt(accounts[i].debit, precision("debit", accounts[i]));
-		tc += flt(accounts[i].credit, precision("credit", accounts[i]));
+		td += flt(accounts[i].debit_in_account_currency, precision("debit", accounts[i]));
```

Comment:
This is not correct. In the case of a multi-currency journal entry, if debit and credit in different currencies, total debit and credit will not match and lead to difference.